.travis.yml

```diff
@@ -79,6 +79,16 @@ _helm_deploy: &helm_deploy
     export release_name_acs=acs-${TRAVIS_BUILD_NUMBER}-${STAGE_SUFFIX}
     export values_file=helm/alfresco-content-services/values.yaml
 
+    if [[ ${TRAVIS_BUILD_STAGE_NAME} == "esc" ]]; then
```

Comment:
We have to define some variables to use inside the postman collection. We need them to skip some tests that we cannot execute using Elasticsearch Connector.